src/components/UploadOption.tsx

```diff
@@ -0,0 +1,19 @@
+import '../styles/UploadVideo.css'
+
+type OptionProp = {
+  imagePath: string
+  title: string
+  alt: string
+  handleClickEvent: (value: void) => void
+}
+
+function UploadOption(props: OptionProp) {
```

Comment:
It is more common to write arrow function:
const UploadOption= (props: OptionProp) => (
    < htmltag></ htmltag>
    )